typescript/web/next-sitemap.js

```diff
@@ -0,0 +1,3 @@
+module.exports = {
+  siteUrl: "https://labelflow.ai",
```

Comment:
Can we at least load the from the .env or something at `build:vercel` time so that:
* it would have a correct value on our preview deployments too
* other users could set their own URL

Reply:
Maybe that there is a Next function which you can call for that? 🤷